games-action/openclonk-7.0

```diff
@@ -0,0 +1,128 @@
+	lib:libX11$secondaryArchSuffix
+	lib:libault$secondaryArchSuffix
+	lib:libat-1.0k$secondaryArchSuffix
+	lib:libc$secondaryArchSuffix
+	lib:libcairo-gobject$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libgcc_s$secondaryArchSuffix
+	lib:libgdk-3$secondaryArchSuffix
+	lib:libgdk_pixbuf-2.0$secondaryArchSuffix
+	lib:libgio-2.0$secondaryArchSuffix
+	lib:libglib-2.0$secondaryArchSuffix
+	lib:libgobject-2.0$secondaryArchSuffix
+	lib:libgthread-2.0$secondaryArchSuffix
+	lib:libgtk-3$secondaryArchSuffix
+	lib:libixml$secondaryArchSuffix
+	lib:libm$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpango-1.0$secondaryArchSuffix
+	lib:libpangocairo-1.0$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libpthread$secondaryArchSuffix
+	lib:librt$secondaryArchSuffix
+	lib:libstdc++$secondaryArchSuffix
+	lib:libthreadutil$secondaryArchSuffix
+	lib:libtinyxml$secondaryArchSuffix
+	lib:libupnp$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+        "
+BUILD_REQUIRES"
+        haiku_devel$secondaryArchSuffix
+	devel:g++$secondaryArchSuffix
+        devel:zlib$secondaryArchSuffix
+        devel:libpng$secondaryArchSuffix
+        devel:libjpeg_turbo$secondaryArchSuffix
+        devel:libfreetype$secondaryArchSuffix
+        devel:libogg$secondaryArchSuffix
+        devel:libvorbis$secondaryArchSuffix
+        devel:libxrandr$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libGLEW$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libX11$secondaryArchSuffix
+	devel:libault$secondaryArchSuffix
+	devel:libat-1.0k$secondaryArchSuffix
+	devel:libc$secondaryArchSuffix
+	devel:libcairo-gobject$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libgcc_s$secondaryArchSuffix
+	devel:libgdk-3$secondaryArchSuffix
+	devel:libgdk_pixbuf-2.0$secondaryArchSuffix
+	devel:libgio-2.0$secondaryArchSuffix
+	devel:libglib-2.0$secondaryArchSuffix
+	devel:libgobject-2.0$secondaryArchSuffix
+	devel:libgthread-2.0$secondaryArchSuffix
+	devel:libgtk-3$secondaryArchSuffix
+	devel:libixml$secondaryArchSuffix
+	devel:libm$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libpango-1.0$secondaryArchSuffix
+	devel:libpangocairo-1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libpthread$secondaryArchSuffix
+	devel:librt$secondaryArchSuffix
+	devel:libstdc++$secondaryArchSuffix
+	devel:libthreadutil$secondaryArchSuffix
+	devel:libtinyxml$secondaryArchSuffix
+	devel:libupnp$secondaryArchSuffix
+	devel:libvorbisfile$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+        "
+BUILD_PREREQUIRES="
+	cmd:cmake
+        cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+        cmd:make
+        "
+BUILD()
+{
+       mkdir -p build
+	cd build
+	cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DDATAROOTDIR=$dataDir \
+		-DENABLE_DESKTOP_ENTRY=OFF -DMANDIR=$manDir -DDOCDIR=$docDir
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
```

Comment:
Use TAB for ident.

Reply:
Don't use spaces, use TAB ...

Reply:
That's fine. `INSTALL()` is OK.
(It's just that for some unknown reason GitHub is not hiding these old messages.)